lang/ghc/Portfile

```diff
@@ -1,16 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           compiler_blacklist_versions 1.0
 
 name                ghc
-# Do not update GHC separate from Haskell Platform.
-# When updating GHC, make sure to revbump all Haskell ports.
-# Also make sure to update the version in the Haskell PortGroup.
-version             7.8.3
-revision            6
+version             8.6.5
 categories          lang haskell
-maintainers         {cal @neverpanic} openmaintainer
+maintainers         {cal @neverpanic ieee.org:s.t.smith @essandess} openmaintainer
```

Comment:
each individual maintainer should have their own {} pair of braces.

Reply:
Thanks done.